src/models/mobileservices/mobilesecurityserviceappcr.js

```diff
@@ -100,7 +100,6 @@ export class MobileSecurityServiceAppCR extends CustomResource {
   }
 
   static getDocumentationUrl() {
-    // TODO: Replace with direct link to documentation when documentation is published
-    return 'https://docs.aerogear.org';
+    return 'https://docs.aerogear.org/aerogear/latest/app-security.html';
```

Comment:
@finp could you confirm that the URL will stay the same or will it be changed from `.../app-security` to `.../mobile-security`?

Reply:
@craicoverflow 
it will be security.html (upstream only)